pages/apis/rest_api/clusters.md

```diff
@@ -243,6 +248,116 @@ Error responses:
 </tbody>
 </table>
 
+## Cluster queues
+
+[Cluster queues](https://buildkite.com/docs/agent/clusters#set-up-a-cluster-set-up-queues) are discrete groups of agents within a cluster. Pipelines in that cluster can target cluster queues to run jobs on agents assigned to those queues.
+
+### Cluster queue data model
+
+<table class="responsive-table">
+  <tbody>
+    <tr><th><code>id</code></th><td>ID of the queue</td></tr>
+    <tr><th><code>graphql_id</code></th><td><a href="/docs/apis/graphql-api#graphql-ids">GraphQL ID</a> of the queue</td></tr>
+    <tr><th><code>key</code></th><td>The queue key</td></tr>
+    <tr><th><code>description</code></th><td>Description of the queue</td></tr>
+    <tr><th><code>url</code></th><td>Canonical API URL of the queue</td></tr>
+    <tr><th><code>web_url</code></th><td>URL of the queue on Buildkite</td></tr>
+    <tr><th><code>cluster_url</code></th><td>API URL of the cluster the queue belongs to</td></tr>
+    <tr><th><code>dispatch_paused</code></th><td>Indicates whether the queue has paused dispatching jobs to associated agents</td></tr>
+    <tr><th><code>dispatch_paused_by</code></th><td>User who paused the queue</td></tr>
+    <tr><th><code>dispatch_paused_at</code></th><td>When the queue was paused</td></tr>
+    <tr><th><code>dispatch_paused_note</code></th><td>The note left when the queue was paused</td></tr>
+    <tr><th><code>created_at</code></th><td>When the queue was created</td></tr>
+    <tr><th><code>created_by</code></th><td>User who created the queue</td></tr>
+  </tbody>
+</table>
+
+### List queues
+
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of a cluster's queues.
+
+```bash
+curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues"
```

Comment:
I think for consistency with our other docs we shouldn't using real IDs in the curl examples:

```suggestion
curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/queues"
```